src/nigui/private/windows/windows.nim

```diff
@@ -35,8 +35,15 @@ const
   ACTCTX_FLAG_RESOURCE_NAME_VALID* = 8
   ACTCTX_FLAG_SET_PROCESS_DEFAULT* = 16
   BN_CLICKED* = 0
+  BM_GETCHECK* = 240.int32
+  BM_SETCHECK* = 241.int32
```

Comment:
Without these set to `int32` building fails when compiling on a Win x64 machine, and probably extends to other areas too but I'm not sure the best way to go about this yet.